packages/frontend/src/components/swap/components/ReviewTransactionDetails.js

```diff
@@ -151,6 +159,60 @@ const ReviewTransactionDetails = ({
         console.log('error parseNearAmount');
     }
 
+    const ratio = () => {
+        const ratio = amountTokenFrom/amountTokenTo;
+        const isAFraction = !!(ratio % 1);
+        return isAFraction
+            ? (ratio).toFixed(5)
+            : removeTrailingZeros((ratio).toString());
+    };
+
+    const isUSN = 
+        (tokenFrom.onChainFTMetadata.name === 'USN' || tokenTo.onChainFTMetadata.name === 'USN') 
+        && CREATE_USN_CONTRACT;
+
+    function transactionDetailsSwitch(token) {
+        switch (token) {
+            case 'USN':
+                return (
+                    <TransactionDetailsUSN
+                        selectedTokenFrom={tokenFrom}
+                        selectedTokenTo={tokenTo}
+                        amount={amountTokenFrom}
+                        exchangeRate={exchangeRate}
+                        tradingFee={tradingFee}
+                        setSlippage={setSlippage}
+                    />
+                );
+            case 'wNEAR':
+                return (
+                    <TransactionDetailsWrappedNear
+                        selectedTokenFrom={tokenFrom}
+                        selectedTokenTo={tokenTo}
+                        estimatedFeesInNear={`${
+                            amountTokenFrom > 1
+                                ? Math.trunc(amountTokenFrom).toString()
+                                : '1'
+                        }`}
+                        estimatedMinReceived={estimatedMinReceived}
+                    />  
+                );
+            default:
```

Comment:
I suggest we collapse this into just the `USN` and the `default` cases and name the component something more generic like `SwapTransactionDetails` wdyt?